### DISCUSSION TITLE
SRT from vMix?

### DISCUSSION THREAD

**Discussion by 3xscreen** [datarhei/restreamer#537] [General]
Has anyone figured out how to send SRT feeds from vMix to Restreamer? I can send SRT from OBS and RTMP/RTMPS from vMix - but I'm having difficulty parsing the Restreamer SRT link in to vMix's SRT Output Settings. Where should the token be added? I thought I could maybe delete the token in Restreamer - but that doesn't seem to work. Any guidance much appreciated.

**Comment by ioppermann** [comment]
The Restreamer SRT link contains the StreamID that you need to fill into the vMix SRT Output Settings. The SRT link looks like this, e.g.: `srt://192.168.1.42:6000/?mode=caller&transtype=live&streamid=channelID,mode:publish,token:abc`. The token is part of the streamid.

This is what you need to fill into the vMix Output Settings (according to the URL above):

Type: `Caller`
Hostname: `192.168.1.42`
Port: `6000`
StreamID: `channelID,mode:publish,token:abc`

You have to adjust the values to your actual SRT link. The passphrase is optional and only required if you actually defined one.

**Reply by 3xscreen** [reply]
Cheers thanks. I'm sorted now 👍
